### ISSUE TITLE
dropdown color

### ISSUE THREAD

**Issue by eloiup** [facultyai/dash-bootstrap-components#220]
theres no color parameter for the dbc.dropdown

i tried changing it through the style parameter

tried both hex color code and "primary". neither worked.

How can one change the color of the dropdown menu?
Is it possible to make it a dbc.dropdown parameter?

ty

**Comment by tcbegley** [comment]
Hey @eloiup 

Thanks for raising this and good spot. I'll make sure this gets added.

**Comment by tcbegley** [comment]
I've opened #221 to address this.

You can try it out by installing [the corresponding prerelease](https://pypi.org/project/dash-bootstrap-components/0.6.3rc1/):

```
pip install dash-bootstrap-components==0.6.3rc1
```

here's a simple example

```python
import dash
import dash_bootstrap_components as dbc

app = dash.Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])

app.layout = dbc.Container(
    dbc.DropdownMenu(
        [dbc.DropdownMenuItem("Item 1"), dbc.DropdownMenuItem("Item 2")],
        label="DropdownMenu",
        color="warning",
        toggle_style={"color": "#1133dd"},
        toggleClassName="p-4",
    ),
    className="p-3",
)

if __name__ == "__main__":
    app.run_server(debug=True)
```

This will make its way into the next full release.

**Comment by eloiup** [comment]
thx!

**Comment by eloiup** [comment]
if i make the dropdown a stand alone comp it works!
if i add it to a navbar, it doesnt!
am i doing something wrong or is it a bug?

`navbar = dbc.NavbarSimple(
    children=[
        dbc.DropdownMenu(
            children=[
                dbc.DropdownMenuItem("item_1", href="/1"),
                dbc.DropdownMenuItem("item_2", href="/2"),
            ],
            nav=True,
            in_navbar=True,
            label="Cargas",
            color="warning",
            toggle_style={"color": "black"},
        ),
        dbc.NavItem(dbc.NavLink("Page 2", href="#")),
        dbc.NavItem(dbc.NavLink("Page 3", href="#")),
    ],
    brand="B8Cargo",
    brand_href="#",
#     color="dark",
#     dark=True,
)`

**Comment by tcbegley** [comment]
The `color` prop is just applying the `btn-{color}` CSS class to the dropdown toggle. When your dropdown is in a navbar this class has no effect. 

You'll notice however that the font color in your example has been set to black as specified in `toggle_style`. You can set additional styles inline or apply a custom class with `toggleClassName`. For example, in your code above, setting

```python
toggle_style={"color": "white", "backgroundColor": "red"}
```

gives me

![image](https://user-images.githubusercontent.com/15220906/59700753-f5482980-91eb-11e9-9e6c-795c254333f6.png)

**Comment by tcbegley** [comment]
I've now merged #221 and released as 0.6.3 so I am going to close this. If the answer above didn't fully address your follow-up question feel free to reopen it.
